integrationtests/tests/fixtures/mod.rs

```diff
@@ -103,6 +104,44 @@ pub struct Fixtures {
     pub task_group: TaskGroup,
 }
 
+/// Iterates through every directory in the `FM_TEST_DIR` and copies the
+/// directories that begin with "db-". These are the databases of the federation
+/// members for the integration test. These databases are backed up to a
+/// supplied directory called `backup_parent`.
+///
+/// Backing up the databases is used to taking a snapshot of the federation
+/// members database state for every integration test. Unit test suites from
+/// newer code versions can then read these databases to verify that database
+/// migrations are working properly.
+///
+/// This function will always copy the databases using the following pattern:
+///
+/// <backup_parent>
+///     test-X
+///         db-A-B
+///         db-C-D
+/// X, B, and D are random numbers. A and C are the PeerId's of the federation
+/// members.
+fn backup_databases(backup_parent: &Path) -> anyhow::Result<()> {
+    let backup_dir = backup_parent.join(format!("test-{}", rng().next_u64()));
+    let test_dir = env::var("FM_TEST_DIR")?;
```

Comment:
Would be probably more composable if this function took `test_dir` as an argument.

Reply:
Removed